crates/tessera-backbone/src/application/policy/mod.rs

```diff
@@ -67,6 +68,31 @@ impl PolicyUseCase for PolicyUseCaseImpl {
 
         return Ok(());
     }
+
+    async fn update(&self, policy_id: &Ulid, new_name: Option<&str>, new_expression: Option<&str>) -> Result<()> {
+        let transaction = self.database_connection.begin_with_organization_scope(&self.workspace_name).await?;
+
+        let mut policy = self
+            .policy_service
+            .get(&transaction, policy_id)
+            .await?
+            .ok_or_else(|| Error::PolicyNotExists { entered_policy_id: policy_id.to_owned() })?;
+
+        if new_name.is_some() || new_expression.is_some() {
```

Comment:
IMO, the conditional statement is redundant here. 

The update logic implicitly requires either `new_name` or `new_expression` to be `Some`, as this is inherent in the function's design. We can simplify the code by removing this check